next.config.js

```diff
@@ -40,6 +40,12 @@ module.exports = {
         destination: '/docs',
         permanent: true,
       },
+      // This path was accidentally used in the announcement email
```

Comment:
I think this will need to be added to the netlify `toml` file, these redirects only work on Vercel

Reply:
Addressed.